error-prone-contrib/src/test/java/tech/picnic/errorprone/bugpatterns/RequestParamTypeTest.java

```diff
@@ -6,6 +6,10 @@
 final class RequestParamTypeTest {
   private final CompilationTestHelper compilationTestHelper =
       CompilationTestHelper.newInstance(RequestParamType.class, getClass());
+  private final CompilationTestHelper restrictedCompilationTestHelper =
```

Comment:
In Error Prone there are cases for both situations; but it can be nice to inline the `restrictedCompilationTestHelper` IMO. Usually there is only one test with the restricted variant and would make it extra clear in the test method itself that we are using extra flags. Besides that, the flag would be closer to the test, which makes the intend clearer as well. 

WDYT?

Reply:
Yes, I've pondered this a few times as well. I'm fine with that, though it sounds like we should do that in a separate PR.

Reply:
I'll file a PR for that then!

Reply:
Filed https://github.com/PicnicSupermarket/error-prone-support/pull/442.